commands/checkout.go

```diff
@@ -74,6 +82,11 @@ func transformCheckoutArgs(args *Args) error {
 		return err
 	}
 
+	if idx := args.IndexOfParam("--no-branch"); idx >= 0 {
```

Comment:
I feel like I shouldn't have to do this. I registered the flag as a `BoolVar` like other commands do, but it wasn't parsing out `--no-branch` automatically.